host_core/lib/host_core/config_plan.ex

```diff
@@ -12,19 +12,18 @@ defmodule HostCore.ConfigPlan do
 
   @impl Vapor.Plan
   def config_plan do
-    wash_config = Path.join(System.user_home(), "/.wash/host_config.json")
-
     [
       %Dotenv{},
       %FilesConfigProvider{
-        paths: [wash_config, "host_config.json"],
+        paths: ["host_config.json"],
```

Comment:
regarding my comment above, I think this is a good change, not reading from `~/.wash/host_config.json` anymore